.idea/allianceauth.iml

```diff
@@ -6,21 +6,16 @@
         <option name="rootFolder" value="$MODULE_DIR$" />
         <option name="settingsModule" value="settings.py" />
         <option name="manageScript" value="manage.py" />
+        <option name="environment" value="&lt;map/&gt;" />
       </configuration>
     </facet>
   </component>
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$" />
-    <orderEntry type="inheritedJdk" />
+    <orderEntry type="jdk" jdkName="Python 2.7.6 (/usr/bin/python2.7)" jdkType="Python SDK" />

```

Comment:
Started out that way too, but quickly got tired of it and figured there must be a better way =P
Anyway, you need the professional version for django support, but it should be available for free if you are a student as I gather you are from your blog.